spec/travel_advice_publisher_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+require "plek"
+require "faker"
+require "date"
+
+describe "travel advice publisher", type: :feature, travel_advice_publisher: true do
+  feature "Creating and deleting new editions" do
+    let(:title) { title_timestamp }
+    let(:summary) { Faker::Lorem.paragraph }
+    let(:part_title) { Faker::Book.title }
+    let(:part_body) { Faker::Lorem.sentence }
+
+    before do
+      delete_existing_draft
```

Comment:
rather than cleaning up previous runs mess before, it'd be cleaner if we could clean up in after block, you might need to use context blocks for it